### PR TITLE
Add more ADO docs

### PR DESCRIPTION
**Pull request by varsanojidan**
## Test plan

docs changes

### CODE REVIEW

**Comment by varsanojidan** [comment]
@indradhanush would you mind adding a section for permissions in doc/admin/external_service/azuredevops.md either in this PR or a follow up PR?

File: doc/admin/external_service/azuredevops.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Azure DevOps
+
+Site admins can sync Git repositories hosted on [Azure DevOps](https://dev.azure.com) with Sourcegraph so that users can search and navigate the repositories.
+
+To connect Azure DevOps to Sourcegraph:
+
+1. Go to **Site admin > Manage code hosts > Add repositories**.
+2. Select **Azure DevOps**.
+3. Next you will have to provide a [configuration](#configuration) for the Gerrit code host connection. Here is an example configuration:
```
Comment (mrnugget) [inline doc/admin/external_service/azuredevops.md:9]:
```suggestion
3. Next you will have to provide a [configuration](#configuration) for the Azure DevOps connection. Here is an example configuration:
```

File: doc/admin/external_service/azuredevops.schema.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
+{
```
Comment (mrnugget) [inline doc/admin/external_service/azuredevops.schema.json:1]: This should be a symlink instead. See the others:

```
φ ls -l doc/admin/external_service/*.schema.json
lrwxrwxrwx    42 thorstenball 21 Jan  2022 doc/admin/external_service/aws_codecommit.schema.json -> ../../../schema/aws_codecommit.schema.json
lrwxrwxrwx    43 thorstenball 21 Jan  2022 doc/admin/external_service/bitbucket_cloud.schema.json -> ../../../schema/bitbucket_cloud.schema.json
lrwxrwxrwx    44 thorstenball 21 Jan  2022 doc/admin/external_service/bitbucket_server.schema.json -> ../../../schema/bitbucket_server.schema.json
lrwxr-xr-x@   34 thorstenball  1 Feb 18:16 doc/admin/external_service/gerrit.schema.json -> ../../../schema/gerrit.schema.json
lrwxrwxrwx    34 thorstenball 21 Jan  2022 doc/admin/external_service/github.schema.json -> ../../../schema/github.schema.json
lrwxrwxrwx    34 thorstenball 21 Jan  2022 doc/admin/external_service/gitlab.schema.json -> ../../../schema/gitlab.schema.json
lrwxrwxrwx    36 thorstenball 21 Jan  2022 doc/admin/external_service/gitolite.schema.json -> ../../../schema/gitolite.schema.json
lrwxr-xr-x    38 thorstenball 14 Jul  2022 doc/admin/external_service/go-modules.schema.json -> ../../../schema/go-modules.schema.json
lrwxr-xr-x    40 thorstenball 14 Jul  2022 doc/admin/external_service/jvm-packages.schema.json -> ../../../schema/jvm-packages.schema.json
lrwxr-xr-x    40 thorstenball 14 Jul  2022 doc/admin/external_service/npm-packages.schema.json -> ../../../schema/npm-packages.schema.json
lrwxrwxrwx    50 thorstenball 21 Jan  2022 doc/admin/external_service/other_external_service.schema.json -> ../../../schema/other_external_service.schema.json
lrwxrwxrwx    36 thorstenball 21 Jan  2022 doc/admin/external_service/perforce.schema.json -> ../../../schema/perforce.schema.json
lrwxrwxrwx    39 thorstenball 21 Jan  2022 doc/admin/external_service/phabricator.schema.json -> ../../../schema/phabricator.schema.json
lrwxr-xr-x    43 thorstenball 14 Jul  2022 doc/admin/external_service/python-packages.schema.json -> ../../../schema/python-packages.schema.json
.rw-r--r--  1.8k thorstenball 28 Oct  2022 doc/admin/external_service/ruby-packages.schema.json
```

(Esxcept `ruby-packages.schema.json`)

**Review by mrnugget — CHANGES_REQUESTED** [review]

File: doc/admin/external_service/index.md [hunk 3]
```diff
@@ -140,7 +151,7 @@ A code host status is:
 - 🔴 _Not available:_ This functionality is not available within Sourcegraph
 
 ## Tier 2: Code Hosts
-We recognize there are other code hosts including CVS, Azure Dev Ops, SVN, and many more. Today, we do not offer native integrations with these code hosts and customers are advised to leverage [Src-srv-git](./non-git.md) and the [explicit permissions API](../permissions/api.md) as a way to ingest code and permissions respectively into Sourcegraph. 
+We recognize there are other code hosts including CVS, SVN, and many more. Today, we do not offer native integrations with these code hosts and customers are advised to leverage [Src-srv-git](./non-git.md) and the [explicit permissions API](../permissions/api.md) as a way to ingest code and permissions respectively into Sourcegraph.
```
Comment (sashaostrikov) [inline doc/admin/external_service/index.md:154]: nice :feelsgood:

**Review by sashaostrikov — COMMENTED** [review]
Looks good currently. There are a couple of fixes pending and also a part of doc from Indra. Will approve as soon as it is done 👍🏻

File: doc/admin/external_service/azuredevops.schema.json (hunk 2 above)
Comment (varsanojidan) [reply doc/admin/external_service/azuredevops.schema.json]: See, that makes a lot more sense 😅.

File: doc/admin/external_service/index.md (hunk 3 above)
Comment (varsanojidan) [reply doc/admin/external_service/index.md:154]: Gotta keep the docs relevant with the times 😛.

**Review by mrnugget — APPROVED** [review]

**Pull request merged**